src/fix.rs

```diff
@@ -0,0 +1,77 @@
+use diesel;
+use diesel::prelude::*;
+use diesel::types::{Integer, VarChar};
+
+use github::{update_issue, update_pr};
+
+use DB_POOL;
+
+pub fn fix() {
+    for (repo, number) in get_duplicate_issues() {
+        delete_duplicate_issues(&repo, number);
+        update_issue(&repo, number).expect("Failed to update issue");
```

Comment:
Hm. Good points. I agree with the ideal world you described. It seems reasonable to me to pass the PgConnection as an argument. That said it seems to me that the effort to do that refactor is probably not worth the payoff of the reliability -- GitHub connectivity issues are rare for the scraper in my experience.

I'd definitely welcome a PR to make things more testable, but unless you'd really like to do that right now I'll just merge this and run it when convenient?